server/routes/visit.ts

```diff
@@ -323,11 +323,19 @@ export default function routes(
           const supportedPrisons = await supportedPrisonsService.getSupportedPrisons(res.locals.user?.username)
           const prisonName = supportedPrisons[visit.prisonId]
 
+          // Current prison phone number configuration, look to move in the future
+          let prisonPhoneNumber = ''
+          if (prisonName === 'Hewell (HMP)') {
```

Comment:
Probably better to test against `visit.prisonId` (e.g. `HEI`) - but that can be addressed when putting stuff in a config file as Paul suggested (filed as [VB-1594](https://dsdmoj.atlassian.net/browse/VB-1594))